test/cppsuite/test_harness/configuration_settings.h

```diff
@@ -39,22 +44,22 @@ class configuration {
         }
     }
 
-    std::string
+    const std::string &
```

Comment:
The function can be const as well.

Reply:
I will make the other functions const as well